controllers/project_controller.js

```diff
@@ -27,6 +27,7 @@ const controller = {
 
   //Method GET: to Show form to edit profile:
   async edit(req, res) {
+    const authenticatedUser = req.session.currentUser;
```

Comment:
Is this required?

Reply:
that line of code is not required any more, because I already set global variable in middelware. 
I will remove this line of code